tests/unit/modules/test_file.py

```diff
@@ -770,7 +790,6 @@ def test_delete_line_in_empty_file(self):
         empty_file.close()
         os.remove(empty_file.name)
 
-
```

Comment:
Can you put this line back in? Then lint will be happy. :)

Reply:
:checkered_flag: